### PR TITLE
flake/dev: add list-plugins script/command

### PR DESCRIPTION
**Pull request by GaetanLepage**
image

### CODE REVIEW

File: flake-modules/dev/devshell.nix [hunk 1]
```diff
@@ -63,6 +63,11 @@
                 ${pkgs.lib.getExe pkgs.python3} -m http.server -d "$doc_derivation"/share/doc
               '';
             }
+            {
+              name = "list-plugins";
+              command = "${pkgs.python3.interpreter} ${./list-plugins.py}";
+              help = "List plugins and get implementation infos";
```
Comment (MattSturgeon) [inline flake-modules/dev/devshell.nix:69]: The help text could try to imply this prints mostly maintenance related info?

**Review by MattSturgeon — APPROVED** [review]
LGTM!

File: flake-modules/dev/devshell.nix (hunk 1 above)
Comment (GaetanLepage) [reply flake-modules/dev/devshell.nix:69]: I thought so and remembered that the devshell was actually made for... well.. development. So I felt that it was not needed.

If you have a better phrasing, I will be happy to use it though !

Comment (MattSturgeon) [reply flake-modules/dev/devshell.nix:69]: I think this is fine :+1:

**Pull request merged**